web/src/components/onboarding/taxId/DisabledTaxId.tsx

```diff
@@ -103,13 +101,9 @@ export const DisabledTaxId = (props: Props): ReactElement => {
   const SimpleDiv = (props: { children: ReactNode }): ReactElement => (
     <div className="flex">
       <div>{props.children}</div>
```

Comment:
Deleted because we have a tool tip in the header in this case and content confirmed we can delete.